src/functions/template-tags/general.php

```diff
@@ -252,14 +252,15 @@ function tribe_get_template_part( $slug, $name = null, array $data = null ) {
 
 if ( ! function_exists( 'tribe_is_ajax_view_request' ) ) {
 	/**
-	 * Check if the current request is for a tribe view via ajax
+	 * Determines if the current request is for a Tribe Events view via AJAX.
```

Comment:
I think we can remove the reference to `Tribe` here. It's been awhile. ;-)
```suggestion
	 * Determines if the current request is for an Events view via AJAX.
```

Reply:
Good point! Already fixed :D